pkg/graph/schema.resolvers.go

```diff
@@ -1907,6 +1932,10 @@ func (r *systemIntakeResolver) CedarSystemID(ctx context.Context, obj *models.Sy
 	return obj.CedarSystemID.Ptr(), nil
 }
 
+func (r *systemIntakeContactResolver) Email(ctx context.Context, obj *models.SystemIntakeContact) (*string, error) {
+	panic(fmt.Errorf("not implemented"))
+}
+
```

Comment:
There's an unimplemented resolver here - I'd make sure to test this query locally!

Reply:
For what it's worth, it's a bit silly that these resolvers need to be implemented at all, because `Email` is just a `string`, but the `gqlgen` script doesn't really know that, so it makes us implement the resolver explicitly.

Reply:
Yep, this needs to be implemented. This should do it:
```
	return (*string)(&obj.Email), nil
```